solidity/contracts/BurntPixArchives.sol

```diff
@@ -69,29 +90,57 @@ contract BurntPixArchives is LSP8CappedSupply {
     )
         LSP8CappedSupply(_maxSupply)
         LSP8IdentifiableDigitalAsset(
-            'Burnt Pix Archives: Season 1',
-            'BURNT1',
+            "Burnt Pix Archives: Season 1",
+            "BURNT1",
             msg.sender,
             _LSP4_TOKEN_TYPE_COLLECTION,
             _LSP8_TOKENID_FORMAT_NUMBER
-        ) {
-            address fractal = address(uint160(uint256(_burntPicId)));
-            address registry = IFractal(fractal).registry();
-            archiveHelpers = _archiveHelpers;
-            burntPicId = _burntPicId;
-            winnerIters = _winnerIters;
-            fractalClone = IArchiveHelpers(_archiveHelpers).createFractalClone(address(this), _codehub, uint256(IRegistry(registry).seeds(fractal)));
-            _setData(_LSP4_CREATORS_ARRAY_KEY, hex"00000000000000000000000000000001");
-            _setData(0x114bd03b3a46d48759680d81ebb2b41400000000000000000000000000000000, abi.encodePacked(_creator));
-            _setData(bytes32(abi.encodePacked(_LSP4_CREATORS_MAP_KEY_PREFIX, hex"0000", _creator)) , hex"24871b3d00000000000000000000000000000000");
-            // royalties
-            _setData(0xc0569ca6c9180acc2c3590f36330a36ae19015a19f4e85c28a7631e3317e6b9d, abi.encodePacked(
+        )
+    {
+        address fractal = address(uint160(uint256(_burntPicId)));
+        address registry = IFractal(fractal).registry();
+        archiveHelpers = _archiveHelpers;
+        burntPicId = _burntPicId;
+        winnerIters = _winnerIters;
+        fractalClone = IArchiveHelpers(_archiveHelpers).createFractalClone(
+            address(this),
+            _codehub,
+            uint256(IRegistry(registry).seeds(fractal))
+        );
+        _setData(
+            _LSP4_CREATORS_ARRAY_KEY,
+            hex"00000000000000000000000000000001"
+        );
+        _setData(
+            //todo: it would be good if we document somewhere internally what these keys represent
+            0x114bd03b3a46d48759680d81ebb2b41400000000000000000000000000000000,
+            abi.encodePacked(_creator)
+        );
+        _setData(
+            bytes32(
+                abi.encodePacked(
+                    _LSP4_CREATORS_MAP_KEY_PREFIX,
+                    hex"0000",
+                    _creator
+                )
+            ),
+            hex"24871b3d00000000000000000000000000000000"
+        );
+
+        // royalties
+        _setData(
+            0xc0569ca6c9180acc2c3590f36330a36ae19015a19f4e85c28a7631e3317e6b9d,
+            abi.encodePacked(
                 hex"001c", // length of data
                 _INTERFACEID_LSP0,
-                _creator,
+                _creator, // TODO: should we separate royalties  reciepients from creator?
```

Comment:
@Ledespana do you mean add another field now in constructor in addition to _creator, that says _royaltyRecipient?